src/amo/components/CollectionManager/index.js

```diff
@@ -288,6 +290,7 @@ export class CollectionManagerBase extends React.Component<
           <input
             onChange={this.onTextInput}
             id="collectionSlug"
+            maxLength={SLUG_MAX_LENGTH}
```

Comment:
I'd suggest moving the constant `SLUG_MAX_LENGTH` in this file instead of `amo/constants`.

Reply:
Done 😄